package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@redpanda-data/docs-extensions-and-macros",
-  "version": "3.10.0",
+  "version": "3.11.0",
```

Comment:
This could be a patch bump. We usually save minors for new extensions or macros.